<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInterface</class>
 <widget class="QWidget" name="StudentInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>165</width>
      <height>139</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="signLessonBut">
       <property name="text">
        <string>Запись на занятие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="futureLessonBut">
       <property name="text">
        <string>Предстоящие занятия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applicationsBut">
       <property name="text">
        <string>Заявки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="historyBut">
       <property name="text">
        <string>История занятий</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>-1</y>
     <width>711</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="signLesson">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>611</width>
       <height>331</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>611</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="studentFinishSignView"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="studentFinishSignBut">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>370</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Записаться</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="futureLesson">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>240</y>
       <width>87</width>
       <height>67</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="studentCancelLessonBut">
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="studentLessonView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>569</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="applications">
    <widget class="QTableView" name="studentAllApplicationsview">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>569</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="history">
    <widget class="QTableView" name="studentHistory">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>569</width>
       <height>469</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
